src/index.js

```diff
@@ -1,11 +1,64 @@
+const EntryType = Object.freeze({
+    Income : 'income',
+    Expense : 'expense',
+});
+
+
 // Entry
-class Entry {}
+class Entry {
+  constructor(date, amount, description) {
+    this.date = date;
+    this.amount = amount;
+    this.description = description;
+  }
+
+  getFormattedAmount() {
+    return `${this.amount} €`;
+  }
+}
 
 // Income
-class Income {}
+class Income extends Entry {
+    constructor(date, amount, description){
+        super(date, amount, description);
+        this.type = EntryType.Income;
+    }
+}
 
 // Expense
-class Expense {}
+class Expense extends Entry {
+    constructor(date, amount, description, paid){
+        super(date, amount, description);
+        this.paid =  paid;
+        this.type = EntryType.Expense;
+    }
+
+    getFormattedAmount(){
+        return '-' + super.getFormattedAmount();
```

Comment:
I love this ❤

Reply:
Did it just in case someone reviewed my code 👌